src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/PKCS1MaskGenerationMethod.cs

```diff
@@ -21,11 +22,15 @@ public string HashName
             set { _hashNameValue = value ?? DefaultHash; }
         }
 
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA5350", Justification = "CryptoConfig.CreateFromName may return platform-dependent objects, so this implementation is limited to SHA-1 for now")]
         public override byte[] GenerateMask(byte[] rgbSeed, int cbReturn)
         {
             using (HashAlgorithm hasher = (HashAlgorithm)CryptoConfig.CreateFromName(_hashNameValue))
```

Comment:
May as well change this from hard cast to soft cast, so asking for AES is a CryptographicException instead of an ICE.

Reply:
Done.